src/attributes/classes.js

```diff
@@ -23,9 +39,7 @@ jQuery.fn.extend( {
 			} );
 		}
 
-		if ( typeof value === "string" && value ) {
-			classes = value.match( rnothtmlwhite ) || [];
```

Comment:
What's important is the difference (in bytes) to what's currently in master. To check that you need to fire `grunt` on `master` and then on your PR.

I've tried to do that but your PR currently fails on `grunt` because of syntax errors. Please make sure to always run `grunt` before pushing changes and running the tests in at least one browser to make sure they still pass.